header/util/util.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <string>
+#include <cassert>
+
+namespace util
+{
+	inline void assertCondition(bool condition, bool onlyInDebug = false)
```

Comment:
Should move the assertCondition  function to src, and we could add a custom error message ? Will look into it